src/credits.html

```diff
@@ -0,0 +1,186 @@
+                    <a class="nav-link" href="index.html">Main Game</a>
+                    <a class="nav-link" href="https://server.sm64js.com/createGame">Custom Game</a>
+                    <a class="nav-link" href="https://vanilla.sm64js.com">Vanilla</a>
+                    <a class="nav-link" href="https://60fps.sm64js.com">60fps</a>
+                    <a class="nav-link" href="https://github.com/sm64js/sm64js">Source Code</a>
+                    <a class="nav-link" href="https://discord.gg/7UaDnJt">Discord Server</a>
+                    <a class="nav-link" href="#Contact">Contact</a>
+                </nav>
+            </div>
+        </header>
+
+        <main role="main" class="inner cover" id="mainContent">
+            <a href="index.html"><h1 class="cover-heading rainbowText mario-font">Super Mario 64 - Javascript</h1></a>
+            <h6>This project is currently still being developed. This page may be live from time to time for testing, but there may be glitches and performance issues. The server will go down and restart at random times without warning. If you would like to help with the project or have feedback please contact me.</h6>
+            <a name="About"></a>
+            <h4>What is this?</h4>
+            <p style="font-size: 20px;">
+                This is an ongoing work-in-progress port of the <a href="https://github.com/n64decomp/sm64">decompilation</a> of original Nintendo game, Super Mario 64, to native Javascript (No Emulation) (No Web Assembly).
+                This project required creating a Javascript WebGL port of <a href="https://github.com/Emill/n64-fast3d-engine">N64 Fast 3D Renderer</a> originally implemented with OpenGL in C. This project also includes the development of online mass multiplayer versions of sm64js and other custom multiplayer game modes.
+            </p>
+            <br />
+
+            <a name="Purpose"></a>
+            <h4>What is the purpose of this port?</h4>
+            <ul style="font-size: 20px;" style="display:inline-block;vertical-align:middle;text-align:left">
+                <li>
+                    <b>Extensibility:</b> More access to more developers to add custom content
+                    <ul style="padding-left:40px">
+                        <li>Rapid Prototyping</li>
+                        <li>More friendly to importing other software modules/packages</li>
+                        <li>More friendly to prototyping multiplayer custom games/mods</li>
+                    </ul>
+                </li>
+                <li><b>Portablity:</b> Sm64 projects can run on any device with a web browser anywhere without needing to download or install anything</li>
+                <li>Opportunity to refactor / rewrite with modern coding style and cleanup</li>
+            </ul>
+            <br />
+            <br />
+
+            <a name="WASM"></a>
+            <h4>What's wrong with Web Assembly / Emscripten</h4>
```

Comment:
it's called WebAssembly.
I know you just moved it, but we should take the opportunity and fix it